ivy/functional/frontends/jax/numpy/linalg.py

```diff
@@ -18,6 +18,11 @@ def eig(a):
     return ivy.eig(a)
 
 
+@to_ivy_arrays_and_back
+def eigvals(a):
```

Comment:
Hi, there is a redefinition of eigvals on Line 49. It seems that it was present before you amde that commit. Further, there is no eigvals function under the Ivy namespace, so your current implementation is incorrect. There is however, an `eigvalsh()` implementation under `ivy/ivy/functional/ivy/linear_algebra.py`. I'd recommend you take a look there and update your function adhering to the standard.